lists/eg.csv

```diff
@@ -674,3 +674,148 @@ http://www.gardennetworks.com,ANON,Anonymization and circumvention tools,2017-02
+https://www.aidsmap.com/HIV-AIDS-services/Egypt/cat/1309/page/1871278/,PUBH,Public Health,2018-02-09,OONI,
+http://www.unaids.org/en/regionscountries/countries/egypt,PUBH,Public Health,2018-02-09,OONI,
+http://www.cairoscene.com/In-Depth/Living-with-HIV-in-Egypt,PUBH,Public Health,2018-02-09,OONI,
+http://www.lovehabibi.com/dating/egyptian-dating/,DATE,Online Dating,2018-02-09,OONI,
+https://dateinegypt.com/,DATE,Online Dating,2018-02-09,OONI,
+https://cairo.dating/,DATE,Online Dating,2018-02-09,OONI,
+http://www.coptic-cairo.com/,CULTR,Culture,2018-02-09,OONI,Minority
+http://minorityrights.org/minorities/copts/,HUMR,Human Rights Issues,2018-02-09,OONI,
+https://en.wikipedia.org/wiki/Persecution_of_Copts,HUMR,Human Rights Issues,2018-02-09,OONI,
+http://www.copts-united.com/,NEWS,News Media,2018-02-09,OONI,
+http://minorityrights.org/minorities/nubians/,HUMR,Human Rights Issues,2018-02-09,OONI,
+https://www.frontlinedefenders.org/en/statement-report/egypt-nubian-human-rights-defenders-trial-peacefully-protesting,HUMR,Human Rights Issues,2018-02-09,OONI,
+http://www.tandfonline.com/doi/full/10.1080/01436597.2016.1206454,HUMR,Human Rights Issues,2018-02-09,OONI,
+https://eipr.org/,HUMR,Human Rights Issues,2018-02-09,OONI,
+https://bahai-egypt.blogspot.com,REL,Religion,2018-02-09,OONI,
+https://bahaieg.org/,REL,Religion,2018-02-09,OONI,
+http://www.bahai.org/persecution/egypt/update,REL,Religion,2018-02-09,OONI,
+http://www.bahai-egypt.org/,REL,Religion,2018-02-09,OONI,
+https://bahaikipedia.org/Egypt,REL,Religion,2018-02-09,OONI,
+http://minorityrights.org/minorities/bahai-of-egypt/,HUMR,Human Rights Issues,2018-02-09,OONI,
+http://horaselthawra.com/,POLR,Political Criticism,2018-02-09,OONI,
+http://www.elghad.com/,POLR,Political Criticism,2018-02-09,OONI,
+http://www.egysdp.com/,POLR,Political Criticism,2018-02-09,OONI,
+https://rsf.org/en/egypt,HUMR,Human Rights Issues,2018-02-09,OONI,
+http://inanities.org/,POLR,Political Criticism,2018-02-09,OONI,One of Egypt's most readable bloggers
+https://suzeeinthecity.wordpress.com/,CULTR,Culture,2018-02-09,OONI,Documented political street art in Cairo throughout the revolution
+https://egyptianchronicles.blogspot.com,POLR,Political Criticism,2018-02-09,OONI,
+https://www.youtube.com/user/sarrahsworld,POLR,Political Criticism,2018-02-09,OONI,
+http://greenarm.org/,ENV,Environment,2018-02-09,OONI,
+http://www.ape.org.eg/,ENV,Environment,2018-02-09,OONI,
+http://web.cedare.org/,ENV,Environment,2018-02-09,OONI,
+http://www.eaee-eg.com/,ENV,Environment,2018-02-09,OONI,
+http://www.ape-egypt.com/,ENV,Environment,2018-02-09,OONI,
+https://bedayaalgbt.wordpress.com/,LGBT,LGBT,2018-02-10,OONI,
+https://solidaritywithegyptlgbt.wordpress.com/,LGBT,LGBT,2018-02-10,OONI,
+https://alborsanews.com/,NEWS,News Media,2018-02-10,OONI,Reportedly blocked
+https://elbadil.com/,NEWS,News Media,2018-02-10,OONI,Reportedly blocked
+https://www.aljazeera.com/,NEWS,News Media,2018-02-10,OONI,Reportedly blocked
```

Comment:
It seems like we have aljazeera.net in the global list. We should consider updating the global list to have the .com as well.